roles/openshift_prometheus/templates/prometheus.j2

```diff
@@ -75,8 +75,9 @@ spec:
 
       - name: prometheus
         args:
-        - --storage.tsdb.retention=6h
-        - --storage.tsdb.min-block-duration=2m
+{% for arg in openshift_prometheus_args %}
+        - {{ arg }}
+{% endfor %}
         - --config.file=/etc/prometheus/prometheus.yml
```

Comment:
the config.file and web.listen are also part of the args may be it will be nicer to add them to ../defaults/main.yaml too ?!

Reply:
As I said in the description: 
`config.file` shouldn't be changed since it is used in other places in the deployment: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/templates/prometheus.j2#L100

`web.listen-addressed` is where oauth-proxy sends upstream data:
https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/templates/prometheus.j2#L57

Reply:
missed that